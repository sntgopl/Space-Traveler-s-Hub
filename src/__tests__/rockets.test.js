import rocketReducer, { cancelRocket, getRocket, reserveRocket } from '../redux/rockets/rockets';

describe('test list defined', () => {
  it('should list of rockets be defined', () => {
    const previousState = [];
    expect(rocketReducer(previousState, getRocket())).toBeDefined();
  });
  it('should reserve of rockets be defined', () => {
    const previousState = [];
    expect(rocketReducer(previousState, reserveRocket())).toBeDefined();
  });
  it('should cancel of rockets be defined', () => {
    const previousState = [];
    expect(rocketReducer(previousState, cancelRocket())).toBeDefined();
  });
});
