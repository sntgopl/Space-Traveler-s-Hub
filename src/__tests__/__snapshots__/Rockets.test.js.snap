// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot should render Rocket Container 1`] = `
<div
  className="rockets"
/>
`;
